### PR TITLE
Record the spent time of each job module

### PR DESCRIPTION
**Pull request by Amrysliu**
See: https://progress.opensuse.org/issues/57419

### CODE REVIEW

File: autotest.pm [hunk 1]
```diff
@@ -407,6 +410,7 @@ sub runalltests {
             $t->save_test_result();
         }
     }
+    bmwqemu::save_json_file(\%modules_spent_time, "modules_spent_time.json") if (%modules_spent_time);
```
Comment (okurz) [inline autotest.pm]: does this save a json result file an additional time? I assume the result file is already saved in another location. Would be good to not save an additional time when this is not necessary.

Comment (Amrysliu) [reply autotest.pm]: yes, those spent times will be saved in a json file. Do you mean I should save the spent time in the `detail-xxxx.json` file?

**Comment by okurz** [comment]
On Monday, 30 December 2019 03.01.34 CET LiuXiaojing wrote:
> Do you mean I should save the spent time in the `detail-xxxx.json` file?

yes, sounds reasonable.

File: autotest.pm (hunk 1 above)
Comment (Martchus) [reply autotest.pm]: Yes, but it is a little bit complicated, see https://github.com/os-autoinst/openQA/pull/2629#pullrequestreview-339219736.

**Comment by Amrysliu** [comment]
it's not a good solution, so close it.

**Comment by Martchus** [comment]
You don't have to close your PRs when a change is required. Just amend and force push your new version.

**Pull request closed without merging**